tests/test_estimateRadialVelocity.py

```diff
@@ -0,0 +1,174 @@
+            self.assertLess(abs(velocity - trueVelocity), permissibleError + 2*velocity_err)
+
+    def createRefSpectrum(self):
+        """
+        Create a fake reference spectrum.
+
+        Returns
+        -------
+        spectrum : `pfs.datamodel.pfsSimpleSpectrum.PfsSimpleSpectrum`
+        """
+        wavelength = np.linspace(300, 1300, 100000)
+
+        peak = self.np_random.uniform(wavelength[0], wavelength[-1], size=200).reshape(-1, 1)
+        depth = self.np_random.beta(1, 5, size=peak.shape)
+        flux = np.prod(1.0 - depth * createRefLsf(wavelength.reshape(1, -1), peak), axis=0)
+
+        target = Target(0, 0, "0,0", 0)
+        mask = np.zeros(len(wavelength), dtype=int)
+        flags = MaskHelper()
+
+        return PfsSimpleSpectrum(target, wavelength, flux, mask, flags)
+
+    def createObsSpectrum(self, refSpectrum, trueVelocity, snr):
+        """
+        Create a fake observed spectrum.
+
+        Parameters
+        ----------
+        refSpectrum : `pfs.datamodel.pfsSimpleSpectrum.PfsSimpleSpectrum`
+            Reference spectrum.
+        trueVelocity : `float`
+            Radial velocity of the observed object.
+        snr : `float
+            Signal-to-noise ratio.
+
+        Returns
+        -------
+        spectrum : `pfs.datamodel.pfsFiberArray.PfsFiberArray`
+        """
+        refWavelength = refSpectrum.wavelength
+        refFlux = refSpectrum.flux
+
+        # Select any 3nm range made of an odd number of sampling points
+        index = np.nonzero((800 <= refWavelength) & (refWavelength <= 803))[0]
+        index = index[: (len(index) + 1) // 2 * 2 - 1]
+        lsfWavelength = refWavelength[index]
+
+        lsfFlux = createLsf(lsfWavelength, lsfWavelength[len(lsfWavelength)//2])
+        lsfFlux /= np.sum(lsfFlux)
+
+        c = 299792.458
+        doppler = np.sqrt((1 + trueVelocity / c) / (1 - trueVelocity / c))
+
+        wavelength = np.linspace(400, 1200, 12000)
+        flux = scipy.signal.fftconvolve(refFlux, lsfFlux, mode="same")
+        flux = scipy.interpolate.interp1d(refWavelength*doppler, flux, kind="cubic")(wavelength)
+
+        variance = flux / (snr*snr)
+        flux = self.np_random.poisson(flux * (snr*snr)) / (snr*snr)
+
+        target = Target(0, 0, "0,0", 0)
+        observations = Observations(
+            np.zeros(0, dtype=int),
+            np.zeros(0, dtype="U0"),
+            np.zeros(0, dtype=int),
+            np.zeros(0, dtype=int),
+            np.zeros(0, dtype=int),
+            np.zeros(shape=(0, 2), dtype=float),
+            np.zeros(shape=(0, 2), dtype=float)
+        )
+        mask = np.zeros(len(wavelength), dtype=int)
+        sky = np.zeros(len(wavelength), dtype=float)
+        covar = np.zeros(shape=(3, len(wavelength)), dtype=float)
+        covar[0, :] = variance
+        covar2 = np.zeros(shape=(0, 0), dtype=float)
+        flags = MaskHelper()
```

Comment:
You should include some masked pixels.

Reply:
Could you suggest the names of flags for this task to observe? I don't know whether the list of all flag names exists or not.

Reply:
Here's a good example: https://github.com/Subaru-PFS/drp_stella/blob/master/python/pfs/drp/stella/photometerLines.py#L44

Reply:
Thank you. I talked with Yamashita-san and he said flux values have been properly interpolated before this task is called, so this task does not need to see masks. I will write such in the docstrings and tell him your link.

Reply:
I don't know how you get observed spectra without masked pixels. I think you need to deal with them.